azure-nextgen-ts-webapp-privateendpoint-vnet-injection/index.ts

```diff
@@ -0,0 +1,174 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as azure_nextgen from "@pulumi/azure-nextgen";
```

Comment:
Other examples use a different import style (e.g. https://github.com/pulumi/examples/blob/master/azure-nextgen-ts-aks/index.ts#L7-L8). Should we do the same here?

Reply:
Fixed.